tests/e2e/tests/regression/database/overview.e2e.ts

```diff
@@ -1,24 +1,16 @@
-import { t } from 'testcafe';
 import { rte } from '../../../helpers/constants';
-import {acceptLicenseTerms, deleteDatabase} from '../../../helpers/database';
+import {acceptLicenseTermsAndAddRECloudDatabase, deleteDatabase} from '../../../helpers/database';
```

Comment:
Let's add spaces before curly brackets the same as for the next import

Reply:
fixed